pkg/core/fee/opcode.go

```diff
@@ -1,199 +1,201 @@
 package fee
 
-import "github.com/nspcc-dev/neo-go/pkg/vm/opcode"
+import (
+	"github.com/nspcc-dev/neo-go/pkg/vm/opcode"
+)
 
-// Opcode returns the deployment prices of specified opcodes.
-func Opcode(opcodes ...opcode.Opcode) int64 {
+// Opcode returns the deployment coefficients of specified opcodes.
+func Opcode(base int64, opcodes ...opcode.Opcode) int64 {
 	var result int64
 	for _, op := range opcodes {
-		result += prices[op]
+		result += coefficients[op]
 	}
-	return result
+	return result * base
 }
 
-var prices = map[opcode.Opcode]int64{
-	opcode.PUSHINT8:     30,
-	opcode.PUSHINT32:    30,
-	opcode.PUSHINT64:    30,
-	opcode.PUSHINT16:    30,
-	opcode.PUSHINT128:   120,
-	opcode.PUSHINT256:   120,
-	opcode.PUSHA:        120,
-	opcode.PUSHNULL:     30,
-	opcode.PUSHDATA1:    180,
-	opcode.PUSHDATA2:    13000,
-	opcode.PUSHDATA4:    110000,
-	opcode.PUSHM1:       30,
-	opcode.PUSH0:        30,
-	opcode.PUSH1:        30,
-	opcode.PUSH2:        30,
-	opcode.PUSH3:        30,
-	opcode.PUSH4:        30,
-	opcode.PUSH5:        30,
-	opcode.PUSH6:        30,
-	opcode.PUSH7:        30,
-	opcode.PUSH8:        30,
-	opcode.PUSH9:        30,
-	opcode.PUSH10:       30,
-	opcode.PUSH11:       30,
-	opcode.PUSH12:       30,
-	opcode.PUSH13:       30,
-	opcode.PUSH14:       30,
-	opcode.PUSH15:       30,
-	opcode.PUSH16:       30,
-	opcode.NOP:          30,
-	opcode.JMP:          70,
-	opcode.JMPL:         70,
-	opcode.JMPIF:        70,
-	opcode.JMPIFL:       70,
-	opcode.JMPIFNOT:     70,
-	opcode.JMPIFNOTL:    70,
-	opcode.JMPEQ:        70,
-	opcode.JMPEQL:       70,
-	opcode.JMPNE:        70,
-	opcode.JMPNEL:       70,
-	opcode.JMPGT:        70,
-	opcode.JMPGTL:       70,
-	opcode.JMPGE:        70,
-	opcode.JMPGEL:       70,
-	opcode.JMPLT:        70,
-	opcode.JMPLTL:       70,
-	opcode.JMPLE:        70,
-	opcode.JMPLEL:       70,
-	opcode.CALL:         22000,
-	opcode.CALLL:        22000,
-	opcode.CALLA:        22000,
-	opcode.ABORT:        30,
-	opcode.ASSERT:       30,
-	opcode.THROW:        22000,
-	opcode.TRY:          100,
-	opcode.TRYL:         100,
-	opcode.ENDTRY:       100,
-	opcode.ENDTRYL:      100,
-	opcode.ENDFINALLY:   100,
+var coefficients = map[opcode.Opcode]int64{
+	opcode.PUSHINT8:     1 << 0,
+	opcode.PUSHINT32:    1 << 0,
+	opcode.PUSHINT64:    1 << 0,
+	opcode.PUSHINT16:    1 << 0,
```

Comment:
While we are here, we may also sort these three as 16 - 32 - 64.While we are here, we may also sort these three as 16 - 32 - 64.

Reply:
Fixed.